### PR TITLE
Add analytics

### LINKED ISSUE #19: Add google analytics

### PR DESCRIPTION
**Pull request by jubar**
Fixes #19

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -7,34 +7,44 @@ module.exports = function(environment) {
...
+      FEATURES: {},
       EXTEND_PROTOTYPES: {
-        // Prevent Ember Data from overriding Date.parse.
         Date: false
       }
     },
 
+    metricsAdapters: [{
+      name: 'GoogleAnalytics',
+      environments: ['development', 'production'],
```
Comment (san650) [inline config/environment.js:18]: Can we remove development? To test this locally we can run `ember serve --prod`.

Comment (san650) [reply config/environment.js:18]: Nevermind, I read the configuration below and we're not actually sending the tracking info to GA 😄

File: config/environment.js [hunk 2]
```diff
@@ -7,34 +7,44 @@ module.exports = function(environment) {
...
     googleFonts: ['Share+Tech+Mono', 'Roboto'],
 
     // Set or update content security policies
     contentSecurityPolicy: {
+      'default-src': "'none'",
       'font-src': "'self' fonts.gstatic.com",
-      'style-src': "'self' fonts.googleapis.com"
+      'style-src': "'self' fonts.googleapis.com",
+      'script-src': "'self' https://www.google-analytics.com/*",
+      'connect-src': "'self' https://www.google-analytics.com/*",
```
Comment (san650) [inline config/environment.js:42]: I think we don't need to add `*` at the end of the URLs. `'self' https://www.google-analytics.com/` is enough.

File: config/environment.js [hunk 3]
```diff
@@ -7,34 +7,44 @@ module.exports = function(environment) {
...
+      'img-src': "'self'",
```
Comment (san650) [inline config/environment.js:43]: Aren't we reading images from github cdn directly?

File: config/environment.js (hunk 2 above)
Comment (san650) [reply config/environment.js:42]: We need to be careful with `script-src` and `connect-src`. If we're using JSON-P, those domains need to appear in `script-src`. If we're doing XHR against any domain, we need to add those to `connect-src`.

**Comment by jubar** [comment]
I paid attention to your comments but I didn't see any error, it is working well on production environment. Please review if works for you and if not please let me know and I'll create tickets. Thanks for your help in this ticket

**Pull request merged**